foundation/foundation-configuration-store/pom.xml

```diff
@@ -13,7 +13,29 @@
     <name>aiSSEMBLE::Foundation::Configuration::Store</name>
     <description>Provides centralized secure storage of configurations</description>
     <packaging>jar</packaging>
+
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-failsafe-plugin</artifactId>
+                    <version>${version.failsafe.plugin}</version>
+                    <configuration>
+                        <systemPropertyVariables>
+                            <KRAUSENING_BASE>${project.basedir}/src/test/resources/configurations/base</KRAUSENING_BASE>
+                            <KRAUSENING_EXTENSIONS>${project.basedir}/src/test/resources/configurations/example-env</KRAUSENING_EXTENSIONS>
+                            <KRAUSENING_PASSWORD>3uQ2j_=wmP5A2q8b</KRAUSENING_PASSWORD>
+                            <version.aissemble.vault>${project.version}</version.aissemble.vault>
```

Comment:
Is this property variable necessary if vault out of the picture?

Reply:
Good point. Its only used in the integration test